<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectNewDialog</class>
 <widget class="QWidget" name="ProjectNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="NW::HeaderLabel" name="label">
     <property name="text">
      <string>Enclosure Project Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NW::Label" name="label_2">
     <property name="text">
      <string>Generates an  Enclosure Project for your Speaker.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="NW::Label" name="label_6">
       <property name="text">
        <string> Volume (l):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="m_pProjectName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ffffff; color: #606060;</string>
       </property>
       <property name="placeholderText">
        <string>Projectname</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="m_pAuthor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ffffff; color: #606060;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="NW::Label" name="m_pInfoLabel">
       <property name="text">
        <string>A Folder with the project name already excist.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="NW::Label" name="label_4">
       <property name="text">
        <string>Select Project Folder: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="NW::Label" name="label_8">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="NW::Label" name="label_7">
       <property name="text">
        <string>Author:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="m_pVolume">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ffffff; color: #606060;</string>
       </property>
       <property name="placeholderText">
        <string>25</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="m_pWarningLabel">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resource/nw.qrc">:/icon/warning_dark.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="m_pProjectPath">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ffffff; color: #606060;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="NW::PushButton" name="m_pProjectFolder">
       <property name="text">
        <string>Ausw√§hlen...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NW::Label</class>
   <extends>QLabel</extends>
   <header>nw/label.hpp</header>
  </customwidget>
  <customwidget>
   <class>NW::HeaderLabel</class>
   <extends>QLabel</extends>
   <header>nw/headerlabel.hpp</header>
  </customwidget>
  <customwidget>
   <class>NW::PushButton</class>
   <extends>QPushButton</extends>
   <header>nw/pushbutton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/nw.qrc"/>
 </resources>
 <connections/>
</ui>
